src/main/java/com/ardaslegends/domain/war/War.java

```diff
@@ -127,6 +133,15 @@ public void addToBattles(Battle battle) {
         addToSet(this.battles, battle, WarServiceException.battleAlreadyListed(battle.getName()));
     }
 
+    public void end() {
+        log.debug("Setting war to inactive");
```

Comment:
It may be useful to add the name of the war in the log statement.

Reply:
Done